### ISSUE TITLE
Pseudo-labeling 

### ISSUE THREAD

**Issue by sard0r** [abanitalebi/simrod#2]
I am using SimROD to generate pseudo labels but the generated pseudo labels are very few almost one pseudo label per image. How can I use it like it is described in the paper? 

I am attaching the pseudo labels I got from SimROD with picture
![Screenshot from 2021-11-11 14-33-33](https://user-images.githubusercontent.com/61651742/141243227-975ba3dd-9f51-4ba5-9f2a-764c2f1231f6.png)
.

It would be helpful for others too I think.

**Comment by abanitalebi** [comment]
Could you elaborate as to what model and source/target data you are using?

**Comment by sard0r** [comment]
! guild run comic:adapt warmup_epochs=100 finetune_epochs=100 cfg=models/yolov5x.yaml output_run_id=${OUTPUT_TEACHER_RUN_DIR} ckpt_run_id=${SRC_TEACHER_MODEL} pseudo_run_id=${SRC_TEACHER_MODEL} gpus=0 bs=12 -y

I am running this command provided in the code. Source is VOC2007/ target my own data.

Is not the model use pretrained weigth of Yolov5 to produce pseudo labels?

Thank you for your help

**Comment by abanitalebi** [comment]
In this case it's adapting from VOC to your custom data.

I can suggest to play with a couple of parameters: I think by default the code uses: "img_size=[416], iou_thres=0.65".
Your example image seems to be high resolution; so you can try increasing the img_size to a higher value such as 1280. Also, lowering the iou_thres increases the number of pseudo-labels. In addition, if you don't care about the source domain data, using COCO is expected to get you a much better source model than VOC.

So in summary: better source model, higher res images, lower threshold.

**Comment by sard0r** [comment]
Which file should I modify dear Author?

**Comment by abanitalebi** [comment]
Please open the code in an IDE and search for those parameters to find their occurrences. 
For example, I see "iou_thres" in "guild.yaml", under "description: Adapt a Pascal VOC model to Comic".
